docs/content/integrations/bigquery/reference.mdx

```diff
@@ -296,11 +310,11 @@ def iris_data() -> pd.DataFrame:
 The dataset will be the last prefix before the asset's name. In this example, the `iris_data` asset will be stored in the `IRIS` dataset, and the `daffodil_data` asset will be found in the `DAFFODIL` dataset.
 
 <Note>
-  The two options for specifying the dataset are mutually exclusive. If you
-  provide <code>dataset</code> configuration to the I/O manager, you cannot also
-  provide it via the asset key and vice versa. If no <code>dataset</code> is
-  provided, either from configuration or asset keys, the default dataset{" "}
-  <code>PUBLIC</code> will be used.
+  The dataset is determined in this order: 1. If the dataset is set via
```

Comment:
Can we put this into a numbered list? It'll make it easier to read.

It could look like this - adding as a code snippet and not a suggestion since `mdx-format` will likely have to be run again:

```
The dataset is determined in this order:
<ol>
<li>If the dataset is set via metadata, that dataset will be used</li>
<li>Otherwise, if there is a <code>key_prefix</code>, that dataset will be used</li>
<li>Otherwise, the dataset set as configuration on the I/O manager will be used</li>
<li>If none of the above are provided, the default dataset will be <code>PUBLIC</code></li>
</ol>
```

Reply:
Just noticed that this is used in the other references too. We can always skip the formatting here to save you some work, and in a follow-up PR work on making some of the shared content in these pages re-usable.

Reply:
it's easy to add, but it would be super nice to template some of these docs out in the future

Reply:
Definitely! This will be a lot easier once we get the new markdown renderer over the finish line.